docs/Config.md

```diff
@@ -116,6 +116,7 @@ git:
   fetchAll: true # Pass --all flag when running git fetch. Set to false to fetch only origin (or the current branch's upstream remote if there is one)
   branchLogCmd: 'git log --graph --color=always --abbrev-commit --decorate --date=relative --pretty=medium {{branchName}} --'
   allBranchesLogCmd: 'git log --graph --all --color=always --abbrev-commit --decorate --date=relative  --pretty=medium'
+  remoteBranchOrder: # one of: 'alphabetical' | 'lastCommit' default is `alphabetical`
```

Comment:
I should've been more clear from the get go, this would be consistent with the rest of the entries:
```
remoteBranchOrder: 'alphabetical' # one of: 'alphabetical' | 'lastCommit'
```

E.g. `notARepository`, `update.method`, `log.showGraph` etc.

Reply:
Oh my bad, should have spotted that myself! Fixed